File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/redis/Tree.java
package org.javamaster.invocationlab.admin.model.redis;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author yudong
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Tree {
    private Integer redisDbIndex;
    private Long keyCount;
    private String label;
    private String labelBase64;
    private Boolean isLeaf;
    private Class<?> labelClass;
    private Integer typeLength;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/redis/Tree.kt
package org.javamaster.invocationlab.admin.model.redis

import org.javamaster.invocationlab.admin.annos.AllOpen

/**
 * @author yudong
 */


@AllOpen
class Tree {
    var redisDbIndex: Int? = null
    var keyCount: Long? = null
    var label: String? = null
    var labelBase64: String? = null
    var isLeaf: Boolean? = null
}
